dns-policy-operator/app/policy/management/commands/create_record_request.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Create record request command."""
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from policy.models import RecordRequest
+
+
+class Command(BaseCommand):
+    """Create Record Request command."""
+    help = 'Create a new record request'
+
+    def add_arguments(self, parser):
+        """Define CLI arguments."""
+        parser.add_argument('host_label', type=str, help='Host label')
+        parser.add_argument('domain', type=str, help='Domain name')
+        parser.add_argument('ttl', type=int, help='TTL')
+        parser.add_argument('record_type', type=str, help='Record type')
+        parser.add_argument('record_data', type=str, help='Record data')
+        parser.add_argument('--requirer_id', type=str, help='Requirer ID', nargs='?', default=None)
+        parser.add_argument('--status', type=str, help='Status', nargs='?', default="pending")
+        parser.add_argument('--status_reason', type=str, help='Status reason', nargs='?', default=None)
+        parser.add_argument('--reviewer', type=str, help='Reviewer username', nargs='?', default=None)
+
+    def handle(self, *args, **options):
```

Comment:
Can't you do `RecordRequest(**options)`
To directly map the dict to the Model ? That would simplify the code and avoid all these intermediary variables.
If not possible, I'd suggest to directly use `options` unless for `reviewer_username` that has some validation

Reply:
I removed the unnecessary variables to directly use `options`